admin/functions-notice.php

```diff
@@ -85,4 +85,84 @@ function aioseop_notice_disable_woocommerce_detected_on_nonpro() {
 		global $aioseop_notices;
 		$aioseop_notices->deactivate_notice( 'woocommerce_detected' );
 	}
+
+	/**
+	 * Set Notice for Disabled Public Blog
+	 *
+	 * Admin Notice when "Discourage search engines from indexing this site" is
+	 * enabled in Settings > Reading.
+	 *
+	 * @since 2.4.2
```

Comment:
shouldn't this (and others below) be since 3.0?

Reply:
Yea it should. Changed